### DISCUSSION TITLE
매일 아침 코드리뷰 & 디스커션 논의하는

### DISCUSSION THREAD

**Discussion by limhyoseok** [DeveloperAcademy-POSTECH/toGather#52] [Ideas]
닐의 의견으로 매일 아침에 코드 리뷰를 진행하기로 했는데
몇가지 더 추가적으로 진행하는 문화를 가지면 좋을 것 같습니다.

- 코드리뷰 진행
- 디스커션 논의하기
- 진행 상황 체크
- - 해야할 것들
- - 각자 맡은 작업 & 예상 종료 시간
- - 업무 할당

단, 코드 리뷰나 디스커션에 너무 많은 시간을 쏟게 되면 개발할 시간이 부족해지니
시간을 정해서 그 시간동안만 하면 좋을 것 같습니다.
(과거 사례 그룹과 레퍼런스... pbxproj 문제...)

**Comment by limhyoseok** [comment]
20220615(수)

**Reply by Sungwooo** [reply]
1. 친구신청 왔을 때 모달알람
2. 앱로고
